src/interfaces/external/ISafe.sol

```diff
@@ -76,6 +76,10 @@ interface ISafe {
 
     function addOwnerWithThreshold(address owner, uint256 _threshold) external;
 
+    function removeOwner(address prevOwner, address owner, uint256 _threshold) external;
+
+    function swapOwner(address prevOwner, address oldOwner, address newOwner) external;
+
```

Comment:
This doesn't work so nicely with my print-state script, since it knows functions in the config contract but not sure we should make decisions based on that.

I can also update the print functionality to also get all functions from contracts in the src folder

Reply:
easy enough to change here – moved it into the config file